sites/all/modules/reol_frontend/reol_frontend.module

```diff
@@ -85,22 +85,40 @@ function reol_frontend_form_search_block_form_alter(&$form, &$form_state, $form_
  * Returns HTML for overlay icons.
  */
 function theme_reol_overlay_icons($variables) {
-  $spans = '';
+  $icon_with_attributes = '';
+
+  $quota_explanation = '';
+  $wrapper_attributes = [
+    'class' => 'overlay-icons'
+  ];
+
 
-  if (!isset($variables['classes']) || !is_array($variables['classes'])) {
-    $variables['classes'] = array();
-  }
-  $variables['classes'][] = 'overlay-icons';
 
   foreach (array_filter($variables['icons']) as $icon) {
-    $spans .= '<span class="icon ' . $icon . '"></span>';
+    $icon_attributes = [
+      'class' => 'icon ' . $icon,
+    ];
+
+    // If material-type is set, then it is of the type that has the material-icon and color-indication of quota
+    // $quota_explanation is for aria-label, so only "visible" for screen-readers
+    if (isset($variables['material_type'])) {
+      $quota_explanation = t('This material is a !type and is on your quota', array('!type' => $variables['material_type']));
+    }
+    // If no_quota is set, then the string changes to not-on-your-quota
+     if (isset($variables['no_quota']) && $variables['no_quota']) {
+       $wrapper_attributes['class'] .= ' no-quota';
+       $quota_explanation = t('This material is a !type and is not on your quota', array('!type' => $variables['material_type']));
+       $icon_attributes['class'] .= ' no-quota';
+   }
   }
-
-  if (!$spans) {
+ 
+  $icon_with_attributes .= '<span aria-label="'. $quota_explanation . '"' . drupal_attributes($icon_attributes) . '></span>';
```

Comment:
Something may be wrong here … You set `$icon_attributes` and `$quota_explanation` inside the `for` loop, but use them outside the loop.

Should `$icon_with_attributes .= …` be moved inside the loop?

Furthermore, `$quota_explanation` may be empty. We probably need a check for this.